helix-term/src/commands.rs

```diff
@@ -5750,3 +5755,156 @@ fn replay_macro(cx: &mut Context) {
         cx.editor.macro_replaying.pop();
     }));
 }
+
+fn goto_label(cx: &mut Context) {
+    jump_to_word(cx, false)
+}
+
+fn extend_to_label(cx: &mut Context) {
+    jump_to_word(cx, true)
+}
+
+fn jump_to(cx: &mut Context, labels: Vec<Range>, extend: bool) {
```

Comment:
```suggestion
fn jump_to_label(cx: &mut Context, labels: Vec<Range>, extend: bool) {
```

what do you think? I'm thinking that just "jump_to" is too generic

Reply:
yeah that seems reasonable , originally `jump_to_word` was actualy named `jump_to_label` so I couldn't call it that originally but I renamed that function after so now that name makes sense